apiv6/localhost_apiserver/main.go

```diff
@@ -1,38 +1,96 @@
-/*
- * Cloud Export Admin API
- *
- * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
- *
- * API version: 202101
- * Generated by: OpenAPI Generator (https://openapi-generator.tech)
- */
-
 package main
 
 import (
 	"flag"
 	"log"
 	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
 
-	cloudexportstub "github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver/go"
+	cloudexportstub "github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver/cloudexport/go"
+	syntheticsstub "github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver/synthetics/go"
 )
 
-func main() {
-	address, storageFilePath := readAddressStorage()
+func Logger(inner http.Handler, name string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		start := time.Now()
+
+		inner.ServeHTTP(w, r)
+
+		log.Printf(
+			"%s %s %s %s",
+			r.Method,
+			r.RequestURI,
+			name,
+			time.Since(start),
+		)
+	})
+}
+
+func makeRouter(cloudexporRepo *cloudexportstub.CloudExportRepo, syntheticsRepo *syntheticsstub.SyntheticsRepo) *mux.Router {
```

Comment:
Please move the function below its usage.

Also:
```suggestion
func makeRouter(cr *cloudexportstub.CloudExportRepo, sr *syntheticsstub.SyntheticsRepo) *mux.Router {
```
to not repeat the name `CloudExportRepo` and decrease verbosity

Reply:
Resolved